src/DotNetNuke.PowerBI/Controllers/SettingsController.cs

```diff
@@ -84,7 +86,8 @@ public ActionResult Index()
                     UserProperty = GetSetting("PowerBIEmbedded_UserProperty","Username"),
                     CustomUserProperty = GetSetting("PowerBIEmbedded_CustomUserProperty",""),
                     CustomExtensionLibrary = GetSetting("PowerBIEmbedded_CustomExtensionLibrary", ""),
-                    ApplicationInsightsEnabled = bool.Parse(GetSetting("PowerBIEmbedded_ApplicationInsightsEnabled", "False"))
+                    ApplicationInsightsEnabled = bool.Parse(GetSetting("PowerBIEmbedded_ApplicationInsightsEnabled", "False")),
+                    BackgroundImageUrl = GetSetting("PowerBIEmbedded_BackgroundImageUrl", "https://user-images.githubusercontent.com/489976/227270883-7a68e527-5ed3-4c02-9ff3-0fb03d2b5af9.png"),
```

Comment:
@alendv better to leave here the default background image url as an empty string to be backward compatible with all the current installations